python/graphstorm/graphpartitioning/array_readwriter/csv.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import pandas as pd
+import pyarrow
+import pyarrow.csv
+
+from .registry import register_array_parser
+
+@register_array_parser("csv")
```

Comment:
what is this for?

Reply:
This is a decorator for this class to register itself for csv files.